cdap-tms/src/main/java/io/cdap/cdap/messaging/client/ClientMessagingService.java

```diff
@@ -404,40 +403,44 @@ private void verifyContentType(Map<String, ? extends Collection<String>> headers
   }
 
   /**
-   * Client side implementation of {@link MessageFetcher}. It streams messages from the server with
+   * Client side implementation of {@link MessageFetchRequest}. It streams messages from the server with
```

Comment:
The comment is not relevant anymore

Reply:
Also I'd jump move fetch to main class. ClientMessageFetcher makes little sense now